subconscious/model.py

```diff
@@ -314,6 +326,25 @@ def get_index_key(cls, column_name):
                 first_iteration = False
             else:
                 result_set = result_set.intersection(temp_set)
+            if order_by:
+                pairs = []
+
+                def _pair_with_zero_score(x):
+                    return [0, x]
+
+                [pairs.extend(_pair_with_zero_score(x)) for x in result_set]
```

Comment:
Nope. That will create a list of list, that is not what we want . The api wants a list with pairs being the consecutive entries.